requirements-docker.txt

```diff
@@ -0,0 +1,16 @@
+numpy
```

Comment:
I think this can be dropped as @endrjuskr split int into requirements/main.txt and extenstions. @endrjuskr can you confirm?